### ISSUE TITLE
Install fails when numpy is not installed

### ISSUE THREAD

**Issue by bodnarbm** [faustomorales/pdqhash-python#3]
Issue
-----

Recently, I was attempting to install pdqhash-python in a fresh virtual environment without having installed any other dependencies first. I encountered an errror when the install process was attempting to execute pdqhash's setup.py.

```
❯ pip install pdqhash
Collecting pdqhash
  Using cached pdqhash-0.2.1.tar.gz (638 kB)
    ERROR: Command errored out with exit status 1:
     command: /home/bodnarbm/.venv/pdqhash-python/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-kown38ok/pdqhash/setup.py'"'"'; __file__='"'"'/tmp/pip-install-kown38ok/pdqhash/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-kown38ok/pdqhash/pip-egg-info
         cwd: /tmp/pip-install-kown38ok/pdqhash/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-kown38ok/pdqhash/setup.py", line 5, in <module>
        import numpy
    ModuleNotFoundError: No module named 'numpy'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Looking at the latest CI builds, I see that it is [also having an initial issue with `numpy` not being installed](https://github.com/faustomorales/pdqhash-python/runs/1401177485#step:6:21), but then `pipenv` reattempts the install after installing the missing dependency.

I can also reproduce this inside of some docker python containers, either from pypi or a local clone of the repo.

**From pypi:**
```
❯ docker run -it python:3.6 pip install pdqhash
Collecting pdqhash
  Downloading pdqhash-0.2.1.tar.gz (638 kB)
     |████████████████████████████████| 638 kB 12.4 MB/s
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-2ops58sm/pdqhash/setup.py'"'"'; __file__='"'"'/tmp/pip-install-2ops58sm/pdqhash/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-g5ngyh5e
         cwd: /tmp/pip-install-2ops58sm/pdqhash/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-2ops58sm/pdqhash/setup.py", line 5, in <module>
        import numpy
    ModuleNotFoundError: No module named 'numpy'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Local repo clone:**

```
❯ docker run -v `pwd`:/opt/pdqhash-python -it python:3.6 pip install /opt/pdqhash-python
Processing /opt/pdqhash-python
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-x20n17b7/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-x20n17b7/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-vfnvcuxw
         cwd: /tmp/pip-req-build-x20n17b7/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-x20n17b7/setup.py", line 5, in <module>
        import numpy
    ModuleNotFoundError: No module named 'numpy'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by bodnarbm** [comment]
I believe that adding support for [PEP-518](https://www.python.org/dev/peps/pep-0518/) using a pyproject.toml file should address this. I can put up a PR that adds pyproject.toml file with the build requirements.

**Comment by faustomorales** [comment]
Thanks, again! This is now fixed on PyPi in v0.2.2.

**Comment by bodnarbm** [comment]
@faustomorales, Thanks for the super prompt response!
